config/rbac/extension_editor_role.yaml

```diff
@@ -0,0 +1,31 @@
+# permissions for end users to edit extensions.
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  labels:
+    app.kubernetes.io/name: clusterrole
+    app.kubernetes.io/instance: extension-editor-role
+    app.kubernetes.io/component: rbac
+    app.kubernetes.io/created-by: operator-controller
+    app.kubernetes.io/part-of: operator-controller
+    app.kubernetes.io/managed-by: kustomize
+  name: extension-editor-role
+rules:
+- apiGroups:
+  - olm.operatorframework.io
+  resources:
+  - extensions
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - update
+  - watch
+- apiGroups:
+  - olm.operatorframework.io
+  resources:
+  - extensions/status
+  verbs:
+  - get
```

Comment:
Nit: Is `get` a valid verb on the CRD status subresource? I'm fairly sure it isn't and this rule accomplishes nothing. (Is this another SDK/Kubebuilder-ism?)

Reply:
Yes, I believe kubebuilder did this, and might put it back...

Reply:
Why would kubebuilder put on an invalid verb for a resource such as this?

Reply:
`client.Status().Get()` does not seem to exist... so perhaps you can't get it.

Reply:
I'll dig in kubebuilder code. But I bet just an uninformed addition to the generation that seemed like a good idea.

Reply:
Wow, it has been there for a very long time (5 years). I suspect it was overlooked originally and is otherwise completely inert so no one has ever noticed.